src/client/src/orangehrmClaimPlugin/components/ClaimActionButtons.vue

```diff
@@ -0,0 +1,226 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+<template>
+  <div
+    :class="{
+      'orangehrm-action-button-container': allowedActions.length < 2,
+      'orangehrm-action-buttons-container': allowedActions.length > 1,
```

Comment:
Less than 2 is 1
Greater than 1 is 2 
is this logic correct? :thinking:

Reply:
less than 2 -> 0,1
greater than 1 -> 2,3,...